sql/kringcoach_toggle.sql

```diff
@@ -0,0 +1,4 @@
+-- Refactor voor kringcoach toggle 27/10/2016
+UPDATE profielen SET kringcoach = '1' WHERE `kringcoach` IS NOT NULL;
+UPDATE profielen SET kringcoach = '0' WHERE `kringcoach` IS NULL;
+ALTER TABLE profielen CHANGE kringcoach kringcoach tinyint(1) NULL DEFAULT NULL;
```

Comment:
Je kunt eenvoudig dit:
```
 		'verticaleleider'	=> array(T::Boolean, true),
		'kringcoach'		=> array(T::Boolean, true),
```
veranderen in dit:
```
 		'verticaleleider'	=> array(T::Boolean),
		'kringcoach'		=> array(T::Boolean),
```
impliceert ``false`` en daarmee NOT NULL.